### PR TITLE
Fbtree

### PR DESCRIPTION
**Pull request by xialiu1988**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: 401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+System.out.println("++++++++++++++++++++++++++");
+        Object[] rr = res.preOrder();
+        for(Object o:rr){
+            System.out.println(o);
+        }
+    }
+
+
+    public static Tree fizzBuzzTree(Node node,Tree tr) {
```
Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java]: I don't think it's necessary to take in a node and tree, I bet you could have figured it out using just one

File: 401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package fizzBuzzTree;
+
+public class App {
```
Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java]: You could have made your method a static method on your already existing tree

Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java]: You could have combined this class with Tree class
